Develop an SQL schema for SOPs

```diff
@@ -1,29 +1,4351 @@
+    psm_search_engine_score_number text,
+    psm_search_engine_score text,
+    smallmolecule_search_engine_score_number text,
+    smallmolecule_search_engine_score text,
+    false_discovery_rate text[],
+    publication_number text,
+    publication text,
+    contact_number text,
+    "contact-name" text,
+    "contact-affiliation" text,
+    "contact-email" text,
+    uri_number text,
+    uri text,
+    fixed_mod_number text,
+    fixed_mod text,
+    "fixed_mod-site" text,
+    "fixed_mod-position" text,
+    variable_mod_number text,
+    variable_mod text,
+    "variable_mod-site" text,
+    "variable_mod-position" text,
+    quantification_method text NOT NULL,
+    "protein-quantification_unit" text,
+    "peptide-quantification_unit" text,
+    "small_molecule-quantification_unit" text,
+    ms_run_number text NOT NULL,
+    "ms_run-format" text,
+    "ms_run-location" text NOT NULL,
+    "ms_run-id_format" text,
+    fragmentation_method_number text NOT NULL,
+    "ms_run-fragmentation_method" text NOT NULL,
+    "ms_run-hash" text,
+    "ms_run-hash_method" text,
+    custom_number text,
+    custom text,
+    sample_number text,
+    species_number text,
+    "sample-species" text,
+    tissue_number text,
+    "sample-tissue" text,
+    cell_type_number text,
+    "sample-cell_type" text,
+    disease_number text,
+    "sample-disease" text,
+    "sample-description" text,
+    "sample-custom" text,
+    assay_number text NOT NULL,
+    "assay-quantification_reagent" text,
+    quantification_mod_number text,
+    "assay-quantification_mod" text,
+    "assay-quantification_mod-site" text,
+    "assay-quantification_mod-position" text,
+    "assay-sample_ref " text,
+    "assay-ms_run_ref" text,
+    study_variable_number text NOT NULL,
+    "study_variable-assay_refs" text NOT NULL,
+    "study_variable-sample_refs" text NOT NULL,
+    "study_variable-description" text NOT NULL,
+    cv_number text NOT NULL,
+    "cv-label" text NOT NULL,
+    "cv-full_name" text NOT NULL,
+    "cv-version" text NOT NULL,
+    "cv-url" text NOT NULL,
+    "colunit-protein" text,
+    "colunit-peptide" text,
+    "colunit-psm" text,
+    "colunit-small_molecule" text,
+    "ID" integer NOT NULL
+);
+
+
+ALTER TABLE "MzTab"."Metadata_Proteomics" OWNER TO postgres;
+
+--
+-- TOC entry 220 (class 1259 OID 17694)
+-- Name: Metadata_Proteomics_Internal_ID_seq; Type: SEQUENCE; Schema: MzTab; Owner: postgres
+--
+
+CREATE SEQUENCE "MzTab"."Metadata_Proteomics_Internal_ID_seq"
+    AS integer
+    START WITH 1
+    INCREMENT BY 1
+    NO MINVALUE
+    NO MAXVALUE
+    CACHE 1;
+
+
+ALTER TABLE "MzTab"."Metadata_Proteomics_Internal_ID_seq" OWNER TO postgres;
+
+--
+-- TOC entry 2966 (class 0 OID 0)
+-- Dependencies: 220
+-- Name: Metadata_Proteomics_Internal_ID_seq; Type: SEQUENCE OWNED BY; Schema: MzTab; Owner: postgres
+--
+
+ALTER SEQUENCE "MzTab"."Metadata_Proteomics_Internal_ID_seq" OWNED BY "MzTab"."Metadata_Proteomics"."ID";
```

Comment:
Good.  We should stick with the convention then of a column called `ID` for the internal sequence

Reply:
done